test/conformance/CMakeLists.txt

```diff
@@ -44,6 +44,23 @@ function(add_conformance_test_with_platform_environment name)
     target_compile_definitions("test-${name}" PRIVATE PLATFORM_ENVIRONMENT)
 endfunction()
 
+function(add_fuzz_test name)
```

Comment:
I wouldn't place these alongside the conformance ones. Let's move them to their own directory.